src/main/java/com/Service/EventService.java

```diff
@@ -0,0 +1,38 @@
+package com.Service;
+
+import com.Model.Event;
+import com.Model.Patient;
+import com.Model.PatientEventAssociation;
+import com.Repository.EventRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
```

Comment:
Not sure this one is really a @Service. I think for this one I prefer to see the code directly in the Controller, since all it's doing is interact with the @Entities.

Reply:
I'm still not sure this service is needed...